model/Individual/Individual.md

```diff
@@ -0,0 +1,24 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Individual
+
+## Summary
+
+A profile to be used when sharing individual SPDX `Element`.
```

Comment:
```suggestion
A profile to be used when sharing individual SPDX Elements.
```
I'm not sure we use backticks anywhere else in the markdown files but so far, it does not seem the standard. I think there was a problem with them not working well with the automated documentation, @maxhbr?

Reply:
I'll remove the back ticks